connector/pom.xml

```diff
@@ -159,6 +159,12 @@
             <version>4.13.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>RELEASE</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Can this be removed?

Reply:
idk i think intellij did that i have no idea how maven works